utils/build/docker/dotnet/parametric/ApmTestClient.csproj

```diff
@@ -13,6 +13,8 @@
   <ItemGroup>
     <PackageReference Include="Datadog.Trace.Bundle" Version="2.31.0" />
     <PackageReference Include="Grpc.AspNetCore" Version="2.52.0" />
+    <PackageReference Include="OpenTelemetry" Version="1.4.0" />
+    <PackageReference Include="LucasP.DuckTyping" Version="0.1.0" />
```

Comment:
Just a thought and not entirely related to this PR, but just wondering if we start using the DuckTyping here in the parametric tests maybe it might make sense to try and publish it from the DataDog repo? Unsure if that is worthwhile/feasible though for this since you already have it copied here

Reply:
Yes, definitely something to consider. It would be very easy to change this line to point to a new Datadog nuget package if we do that.

I think the biggest blocker is that by creating a public repository in the `DataDog` github org and publishing a `Datadog.DuckTyping` nuget package, it will be seen by users as an implicitly supported library.